src/include/rados/librados.hpp

```diff
@@ -1058,6 +1078,8 @@ namespace librados
     // compound object operations
     int operate(const std::string& oid, ObjectWriteOperation *op);
     int operate(const std::string& oid, ObjectReadOperation *op, bufferlist *pbl);
+    int operate_repair_read(const std::string& oid, librados::ObjectReadOperation *o,
+		       uint32_t osdid, int32_t epoch);
     int aio_operate(const std::string& oid, AioCompletion *c, ObjectWriteOperation *op);
```

Comment:
will expose `aio_operate_repair_read()` instead. user is always able to build up his/her own `operate_repair_read()` this method.